tests/unittests/Foundation/NSDataTests.mm

```diff
@@ -97,9 +97,9 @@ void testDecode(NSString* base64String, NSString* expectedDecode, BOOL ignoreUnk
 }
 
 TEST(NSData, Base64DecodeWithOptions) {
```

Comment:
>odeWithOp [](start = 22, length = 9)

This test says `WithOptions`, but it is using the option-free variety.
Should we add another test for the non-option-free variety, which DOES return nil for invalid data?